sql/src/main/java/io/crate/metadata/settings/CrateSettings.java

```diff
@@ -1300,7 +1302,7 @@ public boolean isRuntime() {
     };
 
     public static final StringSetting PSQL_PORT =
-        new StringSetting("port", null, false, "5432-5482", PSQL);
+        new StringSetting("port", null, false, PGSQL_PORT_RANGE, PSQL);
```

Comment:
And please adjust the portrange to +100 to be consistent with http/transport. See also https://github.com/crate/crate/pull/3813#discussion_r70932314